### ISSUE TITLE
VirusTotal Red Flags it 

### ISSUE THREAD

**Issue by Skaveny** [CascadianWorks/Actual-Time-Spent#1]
hi, I just scanned your file with virusTotal and it shows a lot of warnings that it contains Trojans https://www.virustotal.com/gui/file/6049474bf7aa7bf9819508d1598da767642ff20fbc1a717364b6bad542d96432

**Comment by CascadianVR** [comment]
Yes I'm not sure why, i think it has to do with packing it all into a single .exe

If I ever work on this again I'll just bypass combining. Sorry about that.
